libs/editor/libs/editor-common/assets/ZSSRichTextEditor.js

```diff
@@ -1087,8 +1087,11 @@ ZSSEditor.markAllUploadingImagesAsFailed = function(message) {
 
     for(var i = 0; i < matches.size(); i++) {
         var mediaId = matches[i].getAttribute("data-wpid");
+        var videoId = matches[i].getAttribute("data-video_wpid");
         if (mediaId != null) {
```

Comment:
That's a good point. I'm leaning towards checking for empty string instead of using `hasAttribute`: the less DOM methods we can rely on the better, for the same reason of Android WebView differences (by API and by OEM).